### DISCUSSION TITLE
Hi, does anyone know how long it takes for the GitHub Copilot Chat waitlist to be reviewed?"😂

### DISCUSSION THREAD

**Discussion by wenxiayili** [community/community#53058] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

A few days ago, I applied to join the waitlist but have not received any email till now. I want to know how long does it usually take to be reviewed?

**Comment by ghost** [comment, marked as answer]
The hard truth is that you will probably not get an estimate for this.

https://github.com/orgs/community/discussions/50727#discussioncomment-5396477
https://github.com/orgs/community/discussions/50727#discussioncomment-5399532

Although, as noted above, GitHub has been rolling it out within weeks of the announcement. However, the question of a specific time frame depends on too many factors, and probably won't be answered any time soon. It's not directly related but more of an example (as I've said, it depends on too many factors), but I have had some acquaintances who were wait-listed on the original GH Copilot for *a lot* of months and some for far less.

As far as I'm aware, the team has not committed to an official timeline.

**Reply by wenxiayili** [reply]
thanks

**Reply by tedster0629** [reply]
very good. I have learn to see your post. Thank  you.

**Comment by javiervalero31** [comment]
Did you receive the access ?

**Reply by jrcjohn** [reply]
Found this post to see how long I'll be waiting. It's been over a month now

**Comment by zdrjson** [comment]
It's been quite a long time. I have waited for it for about a month.

**Comment by reza8i** [comment]
still no access..

**Comment by asasas234** [comment]
I've been waiting for a long time as well.

**Reply by ChadwickCheng** [reply]
同样在等..

**Reply by lyncodes** [reply]
me too

**Reply by ChadwickCheng** [reply]
Two weeks until I get mail...
![gh_grant](https://github.com/community/community/assets/107307559/993a3df1-8190-4dac-9d5d-89d9ecf402e8)
I've used gh copilot and chat in VSCode for a long time. Time to try jetbrainsO(∩_∩)O

**Comment by kuma0605** [comment]
last day, I joined waitlist. Mark
